### PR TITLE
Add an "Airflow Alert" issue template

### PR DESCRIPTION
**Pull request by AetherUnbound**
## Description

This PR adds an "Airflow Alert" issue template, for use by maintainers when converting Airflow alerts in Slack to GitHub issues. This is modeled off the [Sentry bug issue template](https://github.com/WordPress/openverse-catalog/blob/52acf96b98bf81c7ee20358486f38c6254d91e96/.github/ISSUE_TEMPLATE/sentry_bug.md).

I also made a `🌬️ tooling: airflow` label for these issues.

## Checklist

- [x] My pull request has a descriptive title (not a vague title like
      `Update index.md`).
- [x] My pull request targets the _default_ branch of the repository (`main`) or
      a parent feature branch.
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [ ] I added or updated tests for the changes I made (if applicable).
- [x] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no visible
      errors.
- [ ] I ran the DAG documentation generator (if applicable).

[best_practices]:
  https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/airflow_alert.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+<!-- The link that gets posted in the "Log:" section of the Slack alert -->
+
+> **Note**: _Airflow is currently only accessible to maintainers & those given
+> access. If you would like access to Airflow, please reach out to a
+> maintainer_.
+
+## Description
+
+<!-- Example: We are trying to access property foo of ImportantClass but the instance is null. -->
+<!-- Include any additional information you may have, including potential remedies if any come to mind, and the general context of the code (what causes it to run in the app).  -->
```
Comment (zackkrida) [inline .github/ISSUE_TEMPLATE/airflow_alert.md:19]:
```suggestion
<!-- Include any additional information you may have, including potential remedies if any come to mind, and the general context of the code (what causes it to run in the app).  -->
<!-- Example: We are trying to access property foo of ImportantClass but the instance is null. -->
```

super nit: perhaps describe the section _before_ giving the example.

**Review by zackkrida — APPROVED** [review]
LGTM

**Review by obulat — APPROVED** [review]
Would it make sense to clarify who maintainers are?

File: .github/ISSUE_TEMPLATE/airflow_alert.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+title: "<Replace this with actual title>"
+---
+
+## Airflow log link
+
+<!-- The link that gets posted in the "Log:" section of the Slack alert -->
+
+> **Note**: _Airflow is currently only accessible to maintainers & those given
+> access. If you would like access to Airflow, please reach out to a
+> maintainer_.
```
Comment (zackkrida) [inline .github/ISSUE_TEMPLATE/airflow_alert.md:14]:
```suggestion
> @WordPress/openverse-maintainers_.
```

to @obulat's point.

**Pull request merged**